winlogbeat/beater/winlogbeat.go

```diff
@@ -98,7 +98,7 @@ func (eb *Winlogbeat) init(b *beat.Beat) error {
 		if err != nil {
 			return fmt.Errorf("failed to create new event log: %w", err)
 		}
-		eb.log.Debugf("Initialized EventLog]", eventLog.Name())
+		eb.log.Debugf("Initialized EventLog %s", eventLog.Name())
```

Comment:
How about this in following along with the related changes that broke this (https://github.com/elastic/beats/commit/3d66cc1074a91262704dd739d388e67515e3992d#diff-420d1f0b3025a4724f79acfc014595527b87b1e352cd30dce6dce918d25af023R75).

```suggestion
		eb.log.Debugw("Initialized EventLog", "id", eventLog.Name())
```